index.html

```diff
@@ -358,66 +317,211 @@
-          <code>ArrayBuffer</code></dfn> is an <a><code>ArrayBuffer</code></a>
-          whose data is represented in left-to-right order, row by row top to
-          bottom, starting with the top left, with each pixel's lower 8 bit of
-          16 bit depth value, upper 8 bit of 16 bit depth value, 8 bit reserved
-          data, and another 8 bit reserved data being given in that order for
-          each pixel. Each component of each pixel represented in this array
-          must be in the range 0..255, representing the 8 bit value for that
-          component. The components must be assigned consecutive indices
-          starting with 0 for the top left pixel's lower 8 bit of 16 bit depth
-          value component.
+          A <a>depth map</a> has an associated <dfn>vertical field of
+          view</dfn> which is a double. It represents the vertical angle of
+          view in degrees.
         </p>
+        <p>
+          A <a>depth map</a> has an associated <dfn>format</dfn> which is a
+          DOMString. It represents the format of the <a>depth map</a>, and is
+          one of the following:
+        </p>
+        <dl>
+          <dt>
+            "<code><dfn>linear</dfn></code>"
+          </dt>
+          <dd>
+            TODO: describe
+          </dd>
+          <dt>
+            "<code><dfn>inverse</dfn></code>"
+          </dt>
+          <dd>
+            TODO: describe
+          </dd>
+        </dl>
+        <p>
+          A <a>depth map</a> has an associated <dfn>unit</dfn> which is a
+          DOMString. It represents the unit of the <a>depth map</a> and
+          associated <a>near value</a> and <a>far value</a>. It is one of the
+          following:
+        </p>
+        <dl>
+          <dt>
+            "<code><dfn>mm</dfn></code>"
+          </dt>
+          <dd>
+            Units represented in millimeters (mm).
+          </dd>
+          <dt>
+            "<code><dfn>m</dfn></code>"
+          </dt>
+          <dd>
+            Units represented in meters (m).
+          </dd>
+        </dl>
+        <p>
+          A <a>depth map</a> has an associated <dfn>near value</dfn> which is a
+          double. It represents the minimum range in <a>unit</a>s.
+        </p>
+        <p>
+          A <a>depth map</a> has an associated <dfn>far value</dfn> which is a
+          double. It represents the maximum range in <a>unit</a>s.
+        </p>
+        <dl class="idl" title=
+        " [Constructor(unsigned long sw, unsigned long sh), &nbsp;Constructor(Uint8ClampedArray data, unsigned long sw, optional unsigned long sh)] interface DepthMap">
+        <dt>
+            readonly attribute unsigned long width
+          </dt>
+          <dd>
+            -
+          </dd>
+          <dt>
+            readonly attribute unsigned long height
+          </dt>
+          <dd>
+            -
+          </dd>
+          <dt>
+            readonly attribute Uint16Array data
```

Comment:
I propose to define `data` as `Float32Array` and specify the units is millimeters.